conceptnet5/nodes.py

```diff
@@ -16,52 +16,53 @@
     uri_to_label
 
 
-def standardize_text(text, token_filter=None):
+def standardize_text(text, lang, token_filter=None):
```

Comment:
A few issues here:

- It doesn't seem right to change the function signature of `standardize_text` while keeping the name the same. This seems like it will lead to confusion. I think the function that normalizes and standardizes needs to be a new function.
- The `token_filter` is a separate concern, so maybe it should just be part of `standardized_concept_uri`. Specifying both the language and the `token_filter` is confusing, especially given that the `token_filter` only applies to English, but is not always used in English.
- Other functions in this file take `lang` as an argument that _precedes_ `text`, and the new function signature should match. This is inconsistent with wordfreq, but the reason to do it this way in ConceptNet is that `lang` also precedes `text` when forming URIs.